Services/UI/OJS.Services.Ui.Business/Validation/Implementations/ContestValidationService.cs

```diff
@@ -39,10 +31,11 @@ public async Task<ValidationResult> GetValidationResult((Contest, string, bool,
         }
 
         if (official &&
-            !await this.CanUserCompeteByContestByUserAndIsAdmin(
+            !CanUserCompeteByContestByUserAndIsAdmin(
                 contest,
                 userId,
                 isUserAdmin,
+                isUserLecturerInContest,
                 allowToAdminAlways: true))
```

Comment:
Make this parameter optional in the CanUserCompeteByContestByUserAndIsAdmin method and set true as default

Reply:
Done.